pkg/cmd/operator/cmd.go

```diff
@@ -113,6 +120,11 @@ func NewOperator() *cobra.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
+			caConfigMapPath := filepath.Join(caConfigMapMountPath, caConfigMapName)
+			if _, err := os.Stat(caConfigMapPath); err == nil {
+				go terminateWhenProxyChanges(caConfigMapPath, cancel)
```

Comment:
we already start the fileobserver in a go routine https://github.com/openshift/cloud-credential-operator/pull/320/files#diff-2f87b1f2687fa01b2bd650174a4d17234de24ce7061f469e90c3ea56d72a77ddR232-R236, so why start this function also in go routine?

Reply:
there wasn't any specific need - changed